platform/uwp/export/export.cpp

```diff
@@ -329,7 +329,7 @@ Vector<uint8_t> AppxPackager::make_file_header(FileMeta p_file_meta) {
 	offs += buf_put_int16(0, &buf.write[offs]);
 
 	// File name
-	offs += buf_put_string(p_file_meta.name, &buf.write[offs]);
```

Comment:
This should be kept, if any later `but_put_string` call is added, the `offs` needs to have this data.